apps/examples/testcase/le_tc/network/itc_net_connect.c

```diff
@@ -0,0 +1,274 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/// @file itc_net_connect.c
+/// @brief ITC Scenario Test Case for connect() API
+
+#include <sys/socket.h>
+#include <string.h>
+#include <unistd.h>
+#include <pthread.h>
+#include "tc_internal.h"
+
+#define SERVER_MSG "Hello from server"
+#define CLIENT_MSG "Hello from client 0 "
+#define BUFF_LEN 64
+#define SEC_2 2
```

Comment:
I think that changing SEC_2 to meaningful name would be better.

Reply:
hanges are made as per your comment SECOND_ to WAIT_TIME_SEC_.